src/main/resources/application.yml

```
@@ -38,4 +38,21 @@ spring:
+      mail:
+        smtp:
+          auth: true
+          ssl:
+            enable: true
+          starttls:
+            enable: true
+            required: true
+          connectionTimeout: ${TIMEOUT}
+          timeout: ${TIMEOUT}
+          writeTimeout: ${TIMEOUT}
+        debug: true
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)